src/libServer/JSONConversion.cpp

```diff
@@ -160,8 +160,8 @@ const Json::Value JSONConversion::convertTxBlocktoEthJson(
   // Required by ethers
   retJson["extraData"] = "0x";
   retJson["nonce"] = "0x0";
-  retJson["receiptsRoot"] = "0x";
-  retJson["transactionsRoot"] = "0x0";
+  retJson["receiptsRoot"] = "0x0000000000000000000000000000000000000000000000000000000000000000";
```

Comment:
chainlink unable to parse non hash-length strings, so make them full length. I think this should be fine, no client actually uses these fields I believe